src/asciidoc/docs/customizations/media.adoc

```diff
@@ -0,0 +1,220 @@
+= Media
+:page-slug: media
+:page-description: Standard VCC for selecting an image, file, sound, or video.
+:figure-caption!:
+
+Users can
+//tag::description[]
+select an image, file, sound, or video by selecting from the available options, uploading, or entering a URL.
+//end::description[]
+
+image::mediaVCC.png[title="Media VCC"]
+
+== Sample media VCC
+
+[source,json]
+----
+{
+    "key": "imageOrVideo",
+    "name": "Image or Video",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ["image", "video"],
+        "videoOptions": {
+            "hls": true
+        }
+    }
+}
+----
+
+== Type options
+
+The following code sample shows all the available `typeOptions`.
+Each option is described in the sections that follow the sample.
+
+[source,json]
+----
+{
+    "name": "Media Upload",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ["image", "video", "audio"],
+        "returnType": "url"|"extended",
+        "hideExtensions": false,
+        "videoOptions": {
+            "hls": true,
+            "estimatePoses": false
+        },
+        "imageOptions": {
+            "width": 100,
+        },
+        "audioOptions": {},
+        "fileOptions": {}
+    }
+}
+----
+
+=== acceptOnly
+
+`acceptOnly` allows you to specify the VCC types that are allowed.
```

Comment:
This paragraph needs to be streamlined. I also think we could follow an API-like pattern for describing these typeOptions (make it more like a reference, that is). For example:

"Specifies the underlying media types to allow. The value is an array containing any of the following options...."